test/models/user_test.rb

```diff
@@ -26,6 +27,23 @@
     end
   end
 
+  describe "#time_format" do
+    let(:user) { User.create(name: "jimbob", email: 'test@test.com') }
+    describe 'time_format' do
+      it "should have a default time format of relative" do
+        user.time_format.must_equal('relative')
+      end
+      it "should not update and should raise" do
+        lambda {user.update_attributes!(:time_format => 'foobar') }.must_raise ActiveRecord::RecordInvalid
+      end
+      it "should update" do
```

Comment:
using active forms makes it more readable/simple `it should update` -> `it updates`